Tests/Actions/PatternsSetupTest.php

```diff
@@ -0,0 +1,148 @@
+			'test-pattern-2' => [ 'label' => __( 'Test Pattern 2', 'bcgov_blocks_theme' ) ],
+        ];
+        $expected          = array_merge( $this->get_base_patterns(), $filtered_patterns );
+        Monkey\Functions\when( 'apply_filters' )->justReturn( array_merge( $this->get_base_patterns(), $filtered_patterns ) );
+        $this->assertEquals( $expected, $this->pattern->get_block_pattern_categories() );
+    }
+
+
+    /**
+     * Test the block pattern standard configuration.
+     *
+     * @return void
+     */
+    public function test_Register_Block_Pattern_Standard(): void {
+        $patterns = $this->pattern->get_block_patterns();
+        $this->assertContains( 'header-default', $patterns );
+        $this->assertContains( 'bcgov-detail-card-with-icons', $patterns );
+        $this->assertContains( 'general-hero', $patterns );
+        $this->assertContains( 'bcgov-page-layout-example', $patterns );
+        $this->assertNotContains( 'cleanbc/banners/cleanbc-info-cover-banner', $patterns );
+        $this->assertNotContains( 'cleanbc/active-site-only/cleanbc-actions-toggle-query', $patterns );
+    }
+
+    /**
+     * Test the block pattern standard configuration with option to select all, but not CLEANBC flag.
+     *
+     * @return void
+     */
+    public function test_Register_Block_Pattern_Option_Select_All(): void {
+        Monkey\Functions\when( 'get_option' )->justReturn( 'true' );
+        $patterns = $this->pattern->get_block_patterns();
+        $this->assertContains( 'header-default', $patterns );
+        $this->assertContains( 'bcgov-detail-card-with-icons', $patterns );
+        $this->assertContains( 'general-hero', $patterns );
+        $this->assertContains( 'bcgov-page-layout-example', $patterns );
+        $this->assertContains( 'cleanbc/banners/cleanbc-info-cover-banner', $patterns );
+        $this->assertNotContains( 'cleanbc/active-site-only/cleanbc-actions-toggle-query', $patterns );
+    }
+
+
+    /**
+     * Test the block pattern standard configuration but with filter.
+     *
+     * @return void
+     */
+    public function test_Register_Block_Pattern_With_Filters(): void {
+        $my_patterns = [
+            'test-1',
+            'test-2',
+        ];
+        Monkey\Functions\when( 'apply_filters' )->justReturn( $my_patterns );
+        $patterns = $this->pattern->get_block_patterns();
+        $this->assertContains( 'test-1', $patterns );
+        $this->assertContains( 'test-2', $patterns );
+        $this->assertNotContains( 'general-hero', $patterns );
+        $this->assertNotContains( 'bcgov-page-layout-example', $patterns );
+        $this->assertNotContains( 'cleanbc/banners/cleanbc-info-cover-banner', $patterns );
+        $this->assertNotContains( 'cleanbc/active-site-only/cleanbc-actions-toggle-query', $patterns );
+    }
+
+    /**
+     * Returns expected base patterns.
+     *
+     * @return array
+     */
+    private function get_base_patterns(): array {
```

Comment:
I feel like this and `get_cleanbc_patterns()` could just be constants instead of functions.

Reply:
> I feel like this and `get_cleanbc_patterns()` could just be constants instead of functions.

all cleanbc will eventually be removed from the theme, but yes they could.